data/users.yml

```diff
@@ -902,7 +902,9 @@
   image: blackbell.png
   alt: Blackbell
   use: "Blackbell uses Ember.js for the user interface of webapps / hybrid native apps of small businesses and hotels."
-
+- url: https://noppo.pro
+  alt: Noppo
```

Comment:
Missing the image here :)

Once it's added, email has been received so it should be good to merge!